controllers/survey.controller.js

```diff
@@ -0,0 +1,62 @@
+const DB = require("../configs/db.config");
+const { surveyValidator } = require("../validators/survey.validator");
+
+exports.createSurvey = async (req, res) => {
+  const { error, value } = await surveyValidator(req.body);
+
```

Comment:
@ashabhussan bhai, please try to validate every property for individual `endPoint` if needed.